src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20190613_welder_enabled.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="azmina" id="welder_enabled">
+        <addColumn tableName="CLUSTER">
+            <column name="welderEnabled" type="boolean">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="set welderEnabled to default when welderEnabled is NULL" author="azmina">
```

Comment:
Is this extra changeSet needed? I seem to recall liquibase/mysql automatically setting new non-nullable boolean columns to `false`.